go.mod

```diff
@@ -81,6 +81,7 @@ require (
 	github.com/onsi/gomega v1.6.0 // indirect
 	github.com/opentracing/opentracing-go v1.1.0
 	github.com/polydawn/refmt v0.0.0-20190809202753-05966cbd336a
+	github.com/prometheus/common v0.2.0
```

Comment:
curious how this got added?

Reply:
Whoa.  Yeah, I think goland autoimported it somehow.  I need to turn that off.